src/app/src/main/java/com/cmput301f21t09/budgetprojectname/HabitModel.java

```diff
@@ -0,0 +1,39 @@
+package com.cmput301f21t09.budgetprojectname;
+import java.util.Date;
+
+//TODO: Add Frequency
+public class HabitModel {
+    final private String title;
+    final private String ID;
+    final private String reason;
+    final private int streak;
+    final private Date startDate;
+    private Date lastCompleted;
+
+    HabitModel(String title, String reason, Date startDate) {
```

Comment:
Why does the constructor not have all the possible fields?

Reply:
Oh I was thinking like you could call a method that would increment the streak and set the last completed date?

Reply:
actually w/e I'll just change it